<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code Runner</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
  background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #ff9a9e, stop:0.25 #fad0c4, stop:0.5 #fbc2eb, stop:0.75 #a6c1ee, stop:1 #f6d365);
}

QWidget#centralwidget {
  background-color: rgba(255, 255, 255, 0.9);
  border-radius: 15px;
  margin: 10px 20px 20px 20px;
}

QPushButton {
  background-color: #ff6f61;
  color: white;
  font-size: 18px;
  border-radius: 10px;
  border: none;
}

QPushButton:hover {
  background-color: #ff3b2f;
}

QLineEdit, QTextEdit {
  background-color: #333;
  color: white;
  font-size: 16px;
  padding: 10px;
  border-radius: 5px;
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="code_input">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>540</width>
      <height>250</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your code here...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_console">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>540</width>
      <height>80</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Output will appear here...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
